kafka-ui-react-app/src/components/Connect/Details/__tests__/Details.spec.tsx

```diff
@@ -35,77 +27,96 @@ jest.mock(
 );
 
 describe('Details', () => {
-  containerRendersView(<DetailsContainer />, Details);
-
-  describe('view', () => {
-    const pathname = clusterConnectConnectorPath(
-      ':clusterName',
-      ':connectName',
-      ':connectorName'
-    );
-    const clusterName = 'my-cluster';
-    const connectName = 'my-connect';
-    const connectorName = 'my-connector';
+  const pathname = clusterConnectConnectorPath(
+    ':clusterName',
+    ':connectName',
+    ':connectorName'
+  );
+  const clusterName = 'my-cluster';
+  const connectName = 'my-connect';
+  const connectorName = 'my-connector';
 
-    const setupWrapper = (props: Partial<DetailsProps> = {}) => (
-      <ThemeProvider theme={theme}>
-        <TestRouterWrapper
-          pathname={pathname}
-          urlParams={{ clusterName, connectName, connectorName }}
-        >
-          <Details
-            fetchConnector={jest.fn()}
-            fetchTasks={jest.fn()}
-            isConnectorFetching={false}
-            areTasksFetching={false}
-            connector={connector}
-            tasks={tasks}
-            {...props}
-          />
-        </TestRouterWrapper>
-      </ThemeProvider>
-    );
+  const setupWrapper = (props: Partial<DetailsProps> = {}) => (
+    <Route path={pathname}>
+      <Details
+        fetchConnector={jest.fn()}
+        fetchTasks={jest.fn()}
+        isConnectorFetching={false}
+        areTasksFetching={false}
+        connector={connector}
+        tasks={tasks}
+        {...props}
+      />
+    </Route>
+  );
 
-    it('matches snapshot', () => {
-      const wrapper = create(setupWrapper());
-      expect(wrapper.toJSON()).toMatchSnapshot();
+  it('renders progressbar when fetching connector', () => {
+    render(setupWrapper({ isConnectorFetching: true }), {
```

Comment:
here i see the render is written everywhere , could we just refactor it to the upper function and get rid of the other render , cause it will always do it with render

Reply:
agree, let's schedule it in next iteration of refactoring